src/Apps/Order/Routes/Shipping2/ArtaErrorDialogMessage.tsx

```diff
@@ -0,0 +1,12 @@
+import { RouterLink } from "System/Router/RouterLink"
+
+export const ArtaErrorDialogMessage = () => (
+  <>
+    There was a problem getting shipping quotes. <br />
+    Please contact{" "}
```

Comment:
Thinking I should move this into this file: https://github.com/artsy/force/blob/b7a3a68f306683a8e148b7da498be72370481733/src/Apps/Order/Utils/getErrorDialogCopy.tsx#L7